src/components/Player.vue

```diff
@@ -430,6 +447,49 @@ export default {
   #townsquare:not(.spectator) &.dead .shroud:hover:before {
     opacity: 1;
   }
+
+  .handUp {
+    bottom: 0;
+    left: 0;
+    position: absolute;
+    width: 40%;
+    height: 45%;
+    transform: rotateX(0deg);
+    transform-origin: top left;
+    cursor: pointer;
+    transition: transform 200ms ease-in-out;
+    z-index: 2;
+    pointer-events: none;
+    filter: drop-shadow(0 0 5px rgba(0, 0, 0, 0.8));
+
+    &:before {
+      content: " ";
+      background: url("../assets/handUp.png") center -10px no-repeat;
```

Comment:
Any reason we wouldn't just re-use the same hand icon?

Reply:
(those vue file are a pure trial and error, I've got no idea how to really "read" it, boggles my mind :)